go/vt/vttablet/tabletserver/repltracker/writer.go

```diff
@@ -73,6 +73,11 @@ type heartbeatWriter struct {
 // newHeartbeatWriter creates a new heartbeatWriter.
 func newHeartbeatWriter(env tabletenv.Env, alias topodatapb.TabletAlias) *heartbeatWriter {
 	config := env.Config()
+
+	// config.EnableLagThrottler is a feature flag for the throttler; if throttler runs, then heartbeat must also run
```

Comment:
It is not clear why the changes in this file are required, but I'll trust the tests :)